frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/common/PortalConstants.java

```diff
@@ -330,9 +330,9 @@
 
     // Import credentials
     public static final String SESSION_IMPORT_USER = "importProject_import_user";
-    public static final String SESSION_IMPORT_DB = "importProject_import_db";
     public static final String SESSION_IMPORT_PASS = "importProject_import_pass";
     public static final String SESSION_IMPORT_URL = "importProject_import_url";
+    public static final String IMPORT_PROJECT_NAME = "importProject_import_project_name";
```

Comment:
Is IMPORT_PROJECT_NAME really a import credential or is this constant more like a key which help to specify the the result of the import data?

Reply:
Its only a small recommendation but the sections Import Import Projects, Import credentials, Import Actions, Import Status are not very clearly formatted and listed

Reply:
I change the variable names and string content in this section to make it more coherent. Actually, I was wondering if it would make sense to move all those constants to a new class. Or we have some guidelines about having them in one place?

Reply:
I dont know if we have any guidelines for the naming here but the updated version looks much better!